src/cli/bin.js

```diff
@@ -27,9 +28,23 @@ aliases.forEach((alias) => {
   cli.command(alias.command, alias.describe, alias.builder, alias.handler)
 })
 
-// finalize cli setup
-cli // eslint-disable-line
-  .help()
-  .strict()
-  .completion()
-  .argv
+utils.getIPFS((err, ipfs, cleanup) => {
```

Comment:
init and daemon don't use an `ipfs` instance to run, how does this cover those two cases?